mxcube3/ui/components/Login.jsx

```diff
@@ -1,50 +1,103 @@
+'use strict';
 import ReactDOM from 'react-dom';
-import React from 'react';
+import React, { Component, PropTypes } from 'react'
+import { Redirect, Router, Route } from 'react-router';
 import classNames from 'classnames';
 import "bootstrap-webpack!bootstrap-webpack/bootstrap.config.js";
-import { Nav, Input, ButtonInput } from "react-bootstrap";
+import { Input, ButtonInput, Form, Well } from "react-bootstrap";
+import './Login.css';
+import Main_login from './Main_login'
+import SampleViewContainer from '../containers/SampleViewContainer'
+import SampleGridContainer from '../containers/SampleGridContainer'
+import { Logging } from './Logging'
 
-export default class LoginForm extends React.Component {
+var LoginForm = React.createClass({
 
-    signIn() {
-        let proposal = this.refs.proposal.getValue();
-        let password = this.refs.password.getValue();
-        this.props.signIn(proposal, password);
-    }
+getInitialState: function() {
+    return {
+      login_info: {synchrotron_name: " ", beamline_name: " ", loginType: ""}
+    };
+  },
+  componentDidMount: function(){
+    $.ajax({
```

Comment:
I am not sure but I think getting login info should go to the redux scheme: triggering an action, etc. instead of direct AJAX call ?
Instead of a query, another possibility to retrieve login info may be to have it set by login action through props.